lib/tasks/update_ration_commodity_id.rake

```diff
@@ -0,0 +1,38 @@
+namespace :cats do
+    namespace :ration do
+        desc "update commodity id in ration items with commodity category id"
+        @ration_items= []
+       
+            task update_commodity_on_ration: :environment do
+                puts "Started updating commodity id ...."
+                @total = RationItem.all.count
+                @i = 0
+                RationItem.find_each do |ration_item|
+                
+                @commodity_category_id = Commodity.find_by(id: ration_item.commodity_id)&.commodity_category_id
```

Comment:
Cereal's commodity_category is Food. And when this task runs, it changes all entries of cereal in ration_items in to Food. But Food is not a category, right?